src/main/java/frc/robot/RobotMap.java

```diff
@@ -131,6 +158,8 @@ public static void init()
 
     hatchHolder = new Servo(HATCH_HOLDER_PORT);
 
+    ballIntakeStop = new DigitalInput(BALL_INTAKE_STOP_PORT);
```

Comment:
What is ballIntakeStop?  I'm assuming that it prevents the ball from shooting out the intake mechanism when we're picking up but what hardware component does this correspond to?

Reply:
its a micro switch that triggers when a ball is inside

Reply:
should i change the name to include switch?

Reply:
Name has been changed to include switch